### DISCUSSION TITLE
Update customer channel

### DISCUSSION THREAD

**Discussion by mglenc** [saleor/saleor#13327] [Q&A] [answered]
Hi,
We're working on a B2B platform based on Saleor. Customer management is handled by a separate app sending data to Saleor backend through GraphQL API. We have a channel/warehouse-based setup to limit stock visibility for different customers. Customers are created with `accountRegister` mutation and this is where we specify customer channel. Sometimes we need to change this channel and I was wondering if this is possible through the API. I checked the docs for `accountUpdate` and `customerUpdate` mutations but `channel` field is not mentioned there.

**Comment by timuric** [comment, marked as answer]
channel parameter for customer accounts is used mainly for localization, but not access control, for now the front end only way to limit stock and price visibility. However we are working of functionality to limit exposure of channel information to specific groups of customers to make it more secure

**Reply by mglenc** [reply]
Hi again, I want to ask one more question regarding stock access control. Does the field `accessibleChannels` in `me` query have any effect on stock accessible by customer or does it affect only staff users?
